diagnostic.md

```diff
@@ -5,28 +5,31 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    listr -> listr-list -> listr-list-item
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    component - defines component's behavior
+    template - defines component's markup
     ```
 
 1.  Suppose you have a component '`my-contact`', which is loaded from
     '`app/contacts/template.hbs`' when visiting the `/contacts` path. What is
     the syntax for loading this component inside that template?
 
     ```html
-    <!-- your response here -->
+    {{#each contact.entry as |entry|}}
+      {{entry.name}}
```

Comment:
this is assuming that model data has been passed to contact and contact.entry has multiply entries that we want to call contact.

{{my-contact contact=entry}}